src/components/DpLabel/DpLabel.vue

```diff
@@ -14,25 +14,26 @@
           v-cleanhtml="h" />
       </span>
     </span>
-    <i
+    <dp-contextual-help
       v-if="tooltip !== ''"
-      :class="prefixClass('fa fa-question-circle u-mt-0_125 ml-auto')"
+      :class="prefixClass('mt-px ml-0.5')"
       :aria-label="ariaLabel"
```

Comment:
`aria-label` is already set in DpContextualHelp (with the same value as here), so it can be removed here.

Reply:
done here https://github.com/demos-europe/demosplan-ui/pull/851/commits/71ef394cfeb9ff8470151ba6a8c3a1d29b4ed80b

Reply:
Seems like we can also remove it [in data](https://github.com/demos-europe/demosplan-ui/pull/851/files#diff-65de55f21df8cc95b480ecefa574f35dcf117e8053d810c1a9a13a56ddabb1ccR77-R81), or am i missing something?

Reply:
no, you are right, done https://github.com/demos-europe/demosplan-ui/pull/851/commits/e35402d5656660f196cab7083fd01081768c71d1